components/controls/__tests__/ActionsDropdown-test.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+/* eslint-disable sonarjs/no-duplicate-string */
```

Comment:
My previous comment still hold for this one, is it really necessary ?

Reply:
Yes, it will raise a Code Smell on "renders correctly" being duplicated 3 times. I guess we can close them manually in SQ, but I find that, in tests, we very often get these (like for using `"foo"` or a date `"2019-10-02"`). So sometimes, I just add this comment. It prevents pollution in VSCode, and doesn't raise Code Smells in SQ.

Reply:
BTW, I looked at disabling ESLint rules for certain file patterns, but couldn't find a way. Duplicated strings are very common in our tests, and I'm not in favor of having something like `const foo = "foo";` at the top of a test, just so we don't duplicate that string in certain assertions. I prefer being explicit. But again, VSCode shows warnings, so it's kind of annoying... I personally don't mind such comments, when they're there for a good reason.

Reply:
Yeah I'm not in favor of having `const foo = "foo";` either. I guess eslint comments on top of the files are ok, I just really don't like them in the middle of the code.